Makefile

```
@@ -1,23 +1,24 @@
 REPO_NAME=kubectl-bindrole
 PKGROOT=github.com/Ladicle/kubectl-bindrole
-VERSION ?= $(shell git rev-parse --short HEAD)
+VERSION ?= $(shell git describe --abbrev=0 --tags)
```

In case of supporting both version and git-commit, I think it's more useful not to display the detail error message when the commit is no-version.
like shown below:
```suggestion
VERSION ?= $(shell git describe --abbrev=0 --tags 2>/dev/null || echo no-version)
```